src/battle/16C8E0.c

```diff
@@ -263,6 +263,7 @@ void btl_update(void) {
     s32 cond;
 
     if (battleStatus->inputBitmask != -1) {
+#if !VERSION_PAL
         if ((battleStatus->flags1 & BS_FLAGS1_PARTNER_ACTING) && gGameStatusPtr->multiplayerEnabled) {
```

Comment:
I wonder if this can be achieved with
```suggestion
        if (!VERSION_PAL && (battleStatus->flags1 & BS_FLAGS1_PARTNER_ACTING) && gGameStatusPtr->multiplayerEnabled) {
```

Reply:
I'm not sure how I feel about this - currently, all version-specific stuff can easily be spotted since it's all not indented. this would be breaking that convention

Reply:
Personally I would enjoy breaking that convention. We could go further eg
```c
#if !VERSION_PAL
#define gCurrentLangage LANGUAGE_EN
#endif
```
would remove the need for most instances of
```c
#if VERSION_PAL
switch (gCurrentLanguage) {
    case LANGUAGE_EN:
        foo;
    // ...
}
#else
foo;
#endif
```

Reply:
I think let's hold off on this kind of change for now and revisit version ifdefs when we're matching on everything, like cover talked about in discord

Reply:
Agree